### PR TITLE
Bug 753204 - Discovery --full should be the default

### PR DESCRIPTION
**Pull request by josejulio**
Swapped the default discovery behavior with a --dry-run option and the  --full option.
(The new default is --full).

### CODE REVIEW

**Comment by rhqci** [comment]
Can one of the admins verify this patch?

File: modules/enterprise/agent/src/main/java/org/rhq/enterprise/agent/i18n/AgentI18NResourceKeys.java [hunk 1]
```diff
@@ -1917,15 +1918,13 @@
...
+        + "resources a plugin can actually find.  Note that this will run a detailed server\\n\\\n "
+        + "scan inside the plugin container.\\n\\\n"
+        + "This will update the actual agent inventory by sending an inventory report to the server.\\n\\\n"
+        + "Ignores --plugin, --resourceType and --resourceId unless --dry-run is used.\\n\\\n"
         + "The valid command line arguments are:\\n\\\n"
-        + "\\  -f, --full : Runs a detailed discovery inside the plugin container.\\n\\\n"
-        + "\\               This will update the actual agent inventory by sending\\n\\\n"
-        + "\\               an inventory report to the server.\\n\\\n"
-        + "\\               This ignores --plugin, --resourceType and --resourceId.\\n\\\n"
+        + "\\  -d, --dry-run : Runs a service scan (i.e. it will not try to discover child services\\n\\\n"
```
Comment (pilhuhn) [inline modules/enterprise/agent/src/main/java/org/rhq/enterprise/agent/i18n/AgentI18NResourceKeys.java]: Should that read "server scan" ?

**Comment by pilhuhn** [comment]
Lgtm - @jmazzitelli can you have a look too?

File: modules/enterprise/agent/src/main/java/org/rhq/enterprise/agent/i18n/AgentI18NResourceKeys.java (hunk 1 above)
Comment (josejulio) [inline modules/enterprise/agent/src/main/java/org/rhq/enterprise/agent/i18n/AgentI18NResourceKeys.java]: Yes. My mistake, I'll fix it.

**Pull request merged**